geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -391,17 +418,19 @@ where
     Ok((slope_near_zero, slope_near_infinite))
 }
 
+// Get the infinity line that lies outside of the
+// Delaunay triangle.
 fn get_inf_outside_triangle<T: GeoFloat>(
     triangle: &Triangle<T>,
-    circumcenter: &Coord<T>,
+    circumcentre: &Coord<T>,
     midpoint: &Coord<T>,
     slope_threshold: Option<T>,
 ) -> Result<Line<T>>
 where
     f64: From<T>,
 {
     let two = T::from(2.).ok_or(VoronoiDiagramError::CannotConvertBetweenGeoGenerics)?;
-    let tmp_line = Line::new(*midpoint, *circumcenter);
+    let tmp_line = Line::new(*midpoint, *circumcentre);
```

Comment:
`tmp_line` needs a better name

Reply:
re the comment a bit further down "If the infinity line crosses one of the other lines of the triangle ... the tmp_line needs to be flipped." looks like it is describing the majority of the function, and could be put in the `fn`s docstring.

Reply:
I changed this to `inf_line`